### PR TITLE
Feature/first quest enhancments

### PR DESCRIPTION
**Pull request by amaredeus**
## 1.1.0-RELEASE (2021-11-26)

1. Handle edge cases for first quest
   - flow captcha in text channel only
   - add first quest start command
   - enhance stability

### CODE REVIEW

File: src/app/utils/ServiceUtils.ts [hunk 1]
```diff
@@ -295,6 +295,23 @@ const ServiceUtils = {
 		};
 		return (await dmChannel.awaitMessages(replyOptions)).first().attachments.first();
 	},
+	
+	sendOutErrorMessage: async (ctx: CommandContext): Promise<any> => {
```
Comment (nonsensecodes) [inline src/app/utils/ServiceUtils.ts:299]: In a future update, can we include the support server link here?

File: src/app/events/first-quest/StartFirstQuestFlow.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import fqConstants from '../../service/constants/firstQuest';
```
Comment (tikisailor) [inline src/app/events/first-quest/StartFirstQuestFlow.ts:1]: should be import firstQuest from '../service/constants/firstQuest';

Comment (tikisailor) [reply src/app/events/first-quest/StartFirstQuestFlow.ts]: '../../service/constants/firstQuest'

File: src/app/commands/first-quest/FirstQuest.ts [hunk 3]
```diff
@@ -1,25 +1,31 @@
...
 } from 'slash-create';
-import roleIds from '../../service/constants/roleIds';
 import ServiceUtils from '../../utils/ServiceUtils';
 import ConfigureFirstQuest from '../../service/first-quest/ConfigureFirstQuest';
 import ValidationError from '../../errors/ValidationError';
 import discordServerIds from '../../service/constants/discordServerIds';
 import { LogUtils } from '../../utils/Log';
 import FirstQuestPOAP from '../../service/first-quest/FirstQuestPOAP';
+import { sendFqMessage, switchRoles } from '../../service/first-quest/LaunchFirstQuest';
+import fqConstants from '../../service/constants/firstQuest';
```
Comment (tikisailor) [inline src/app/commands/first-quest/FirstQuest.ts:14]: should be import firstQuest from '../../service/constants/firstQuest';

Are there mmore instances of this in other files?

Comment (tikisailor) [inline src/app/commands/first-quest/FirstQuest.ts:14]: should be import firstQuest from '../../service/constants/firstQuest';

Are there mmore instances of this in other files?

**Review by nonsensecodes — APPROVED** [review]
Looks good! Thank you, @amaredeus & @tikisailor!

File: src/app/utils/ServiceUtils.ts (hunk 1 above)
Comment (amaredeus) [reply src/app/utils/ServiceUtils.ts:299]: already there ser on line 306 :)

**Review by tikisailor — APPROVED** [review]

**Pull request merged**